bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -90,6 +91,41 @@ class BlockIssuerFeature extends Feature {
     }
 }
 
+/**
+ * Block Issuer feature.
+ */
+class StakingFeature extends Feature {
+    /**
+     * The amount of coins that are locked and staked in the containing account.
+     **/
+    readonly stakedAmount: bigint;
```

Comment:
`bigint` instead of the type alias `u64` is maybe better for the user to be directly aware of the type, we might want to not use `u64` anymore because it can be confusing to the user as @kwek20 hinted to if I'm not mistaken.

Reply:
Not fully sure to understand, these values are internally (rust) u64, so we should probably use u64? Even though in JS u64 point to bigint, these are not bingints.

Reply:
yes, but when people want to create the value, they have to use the `bigint` constructor and `typeof value` would be `bigint`

Reply:
Oh alright. Is there no other way to define `u64` than a simple typedef? A more strongly typed alternative? Otherwise these typedef seems a bit useless to me...

Reply:
The type alias is just an indication that you can't use negative values or values bigger than the max of u64, but it doesn't enforce anything. I don't think we have any good alternative to using just `bigint` as it has many advantages, for example you can use normal arithmetic operation on bigint values. but if you wrap it somehow, you can't do that anymore.

Reply:
Are we changing these then?